README.md

```diff
@@ -156,3 +156,38 @@ To list all installed versions of the module available for import:
 ```powershell
 Get-Module -Name Compile-SourceScript -ListAvailable -Verbose
 ```
+
+## Common errors
+
+Here are some very common errors seen when using this module or the compilers (`amxxpc` or `spcomp`).
+
+#### Error: `No such file or directory`
```

Comment:
Removing code block might make titles more readable

Reply:
It might. Except the error messages should be in code blocks. The new GitHub formatting for inline code doesn't look good @leojonathanoh

Reply:
@joeltimothyoh generally when code blocks are used in titles, they make titles appear less like a titles.

Reply:
It would help if the content was organized with minimal redundancy.

Reply:
will try

Reply:
fixed